moonshot/data/runners-modules/redteaming.py

```diff
@@ -77,13 +79,14 @@ async def generate(
         self.database_instance = database_instance
         self.session_metadata = session_metadata
         self.red_teaming_type = red_teaming_type
+        self.red_teaming_progress = red_teaming_progress
 
         if self.red_teaming_type == RedTeamingType.AUTOMATED:
             print("[Red Teaming] Starting automated red teaming...")
             await self.run_automated_red_teaming()
         elif self.red_teaming_type == RedTeamingType.MANUAL:
             print("[Red Teaming] Starting manual red teaming...")
-            await self.run_manual_red_teaming()
+            return await self.run_manual_red_teaming()
```

Comment:
just checking the return here is intented? the automated one didnt have a return

Reply:
yeah for now. automated rt will update the chats via callback while manual rt will update the chats by returning a list of dict